.env.example

```diff
@@ -1 +1,3 @@
-DATABASE_URL="postgresql://databaseusername:databasepassword@localhost:5432/mydb?schema=public"
\ No newline at end of file
+DATABASE_URL="postgresql://databaseusername:databasepassword@localhost:5432/mydb?schema=public" # README.md's section "Setting the environment variables" will help with this
+POSTGRES_USERNAME="databaseusername" # Postgres database's username. It is same as DATABASE_URL's databaseusername
+POSTGRES_PASSWORD="databasepassword" # Postgres database's password. It is same as DATABASE_URL's databasepassword
```

Comment:
Mihin näitä kahta riviä tarvitaan?

Reply:
Hyvä löytö! En löytänyt mitään käyttöä koodista noille kahdelle riville ja testasin kanssa, että tietokanta toimii ilman noitakin, joten poistoon lähti! :+1:

Luultavasti liittyi jollain tavalla NextAuthiin, koska kopioin tämän `.env.example`-tiedostonsen vanhasta branchista :sweat_smile: